cluster-provision/k8s/scripts/provision.sh

```diff
@@ -37,14 +37,7 @@ repo_gpgcheck=1
 gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
        https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
 EOF
-yum install -y \
-  docker-common-1.13.1-75.git8633870.el7.centos.x86_64 \
-  origin-docker-excluder-3.11.0-1.el7.git.0.62803d0.noarch \
-  python-docker-py-1.10.6-4.el7.noarch \
-  docker-client-1.13.1-75.git8633870.el7.centos.x86_64 \
-  cockpit-docker-176-2.el7.centos.x86_64 \
-  docker-1.13.1-75.git8633870.el7.centos.x86_64 \
-  python-docker-pycreds-1.10.6-4.el7.noarch
+yum install -y docker
```

Comment:
without this change the provision is broken because this packages doesn't exist anymore.